build.gradle

```diff
@@ -1,23 +0,0 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
-
-buildscript {
-    repositories {
```

Comment:
why have you removed this all?
You don't need to send this file in pull request.